client/client-multi/client-cpp/impl/channel/impl/DefaultOperationTcpChannel.cpp

```diff
@@ -369,10 +369,12 @@ boost::system::error_code DefaultOperationTcpChannel::sendPingRequest()
 
 void DefaultOperationTcpChannel::readFromSocket()
 {
-    boost::asio::async_read(*sock_, *responseBuffer_,
-              boost::asio::transfer_at_least(1),
-              boost::bind(&DefaultOperationTcpChannel::onReadEvent, this,
-                      boost::asio::placeholders::error));
+    boost::asio::async_read(*sock_
+                          , *responseBuffer_
+                          , boost::asio::transfer_at_least(1)
+                          , boost::bind(&DefaultOperationTcpChannel::onReadEvent
+                                      , this
+                                      , boost::asio::placeholders::error));
```

Comment:
Don't put commas and start of line. Sorry, it wasn't stated in the guide; I'll try to update it soon.